shared/gast_to_code/gast_to_code_router.py

```diff
@@ -13,6 +13,9 @@ def gast_to_code(gast, out_lang, lvl=0):
     converter = ConverterRegistry.get_converter(out_lang)
     
     if type(gast) == list:
+        # bash has no comma between function arguments
+        if out_lang == "bash":
+            return general_helpers.list_helper(gast, out_lang, " ")
```

Comment:
Would be nice if this "list delimiter" was provided by the language instead of it needing to be known to the generic router.

Reply:
+1